instances/latest/terminologies/UBERONParcellation/instances/latest/terminologies/UBERONParcellation/paraventricularNucleusOfTheHypothalamus.jsonld

```diff
@@ -0,0 +1,29 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/paraventricularNucleusOfTheHypothalamus",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+  "definition": "'Paraventricular nucleus of the hypothalamus' is a nucleus in the anterior part of the hypothalamus.",
```

Comment:
Change to:
`Is a hypothalamic nucleus. Is part of the periventricular zone of hypothalamus. [auto-generated from properties of the [UBERON ontology term](http://purl.obolibrary.org/obo/UBERON_0001930) ('is_a' and 'relationship')]`